aiidalab/app.py

```diff
@@ -124,6 +129,11 @@ def __init__(self, name, app_data, aiidalab_apps_path):
         self.refresh_async()
         self._watch_repository()
 
+    def __repr__(self):
+        app_data_argument = None if self._registry_data is None else asdict(self._registry_data)
+        return f"AiidaLabApp(name={self.name!r}, app_data={app_data_argument!r}, " + \
+                f"aiidalab_apps_path={os.path.dirname(self.path)!r})"
```

Comment:
Two suggestions in one:
- Wrap the representation in the "standard" `<>`.
- Write the return string value without the use of string concatenation and `\`.
```suggestion
        return (
            f"<AiidaLabApp(name={self.name!r}, app_data={app_data_argument!r}, "
            f"aiidalab_apps_path={os.path.dirname(self.path)!r})>"
        )
```